src/components/LenderProfile/LenderBadges.vue

```diff
@@ -1,33 +1,37 @@
 <template>
-	<section v-if="completedAchievements.length > 0">
-		<h4
-			v-if="!isLoading"
-			class="data-hj-suppress"
-		>
-			{{ badgesTitle }}
-		</h4>
-		<kv-loading-placeholder
-			v-else
-			style="height: 30px; width: 250px;"
-		/>
-		<badges-list
-			class="tw-my-4"
-			:completed-achievements="completedAchievements"
-			:total-possible-badges="totalPossibleBadges"
-			:is-loading="isLoading"
-		/>
-	</section>
+	<async-lender-section @visible="isLoading = false;">
```

Comment:
I think these need to be `() => isLoading = false` for the Vue migration.

Reply:
Gotcha. Just curious where is the vue3 spec for this.

Reply:
It was just something I ran into during the migration.